python/lsst/ts/observatory/control/data/atcalsys_schema.yaml

```diff
@@ -4,48 +4,105 @@ title: Configuration file validation schema of Auxiliary Telescope Calibration S
 type: object
 properties:
   calib_type:
+    description: >-
+      Type of flat field calibration. There are two types, `Whitelight` for
+      broad spectrum light and `Mono` for monochromatic.
     type: string
     enum:
       - Mono
       - WhiteLight
   use_camera:
+    description: Indicates whether the operation requires a camera.
     type: boolean
   atspec_filter:
+    description: Name of the filter to select for the spectrograph.
     type: string
   atspec_grating:
+    description: Name of the grating to select for the spectrograph.
     type: string
   wavelength:
+    description: >-
+      Center wavelength value in nm, used for configuring the monochromator
+      output.
     type: number
   wavelength_width:
+    description: >-
+      Optional. Defines the width of the wavelength scan range to configure the
+      monochromator for the flat-field calibration sequence when using
+      monochromatic light.
     type: number
   wavelength_resolution:
+    description: >-
+      Optional. When using a monochromatic light source, it defines the
+      resolution used to sample values within the scan range, which has a width
+      defined by `wavelength_width` and is centered around the `wavelength`
+      attribute.
     type: number
   monochromator_grating:
+    description: >-
+      Select the grating of the monochromator that will be in the path of
+      light.
     type: string
     enum:
       - MIRROR
       - RED
       - BLUE
   exit_slit:
+    description: >-
+      Exit slit width for the monochromator, from 0 to 7 mm (inclusive).
     type: number
   entrance_slit:
+    description: >-
+      Entrance slit width for the monochromator, from 0 to 7mm (inclusive).
     type: number
   electrometer_integration_time:
+    description: >-
+      Integration time in seconds (166.67e-6 to 200e-3) for each sample.
+
+      The integration time (measurement speed) of the analog to digital (A/D)
+      converter, the period of time the input signal is measured (also known as
+      aperture). Due to the time it takes to read the buffer and process the
+      data, this is not the rate at which samples are taken. This is generally
+      specified by the Power Line Cycle (PLC), where 1 PLC for 60Hz is
+      16.67msec. Fast integration=0.01PLC; Medium integration=0.1PLC; Normal
+      (default) integration = 1PLC; High Accuracy integration=10PLC. Here the
+      integration is set in seconds.
```

Comment:
Here I'm including a lot of information. I guess it could be more concise. Let me know if the extra paragraph should be summarized or removed. I have taken it from the command specification.